src/js/core/widget/core/Popup.js

```diff
@@ -630,8 +630,61 @@
 					// set state of popup
 					self.state = states.CLOSED;
 				}
+
+				if (self._ui.content.scrollHeight > self._ui.content.clientHeight) {
+					self._ui.footer.classList.add("bottomDivider");
+				}
 			};
 
+			prototype._onScroll = function (content) {
+				// content.addEventListener("scroll", function () {
+				var self = this;
+
+				eventUtils.trigger(content, "scrollstart");
+
+				if (content.scrollTop === 0) {
+					self._ui.header.classList.remove("topDivider");
+					self._ui.footer.classList.add("bottomDivider");
+				} else if (content.scrollHeight - content.clientHeight === content.scrollTop) {
+					self._ui.header.classList.add("topDivider");
+					self._ui.footer.classList.remove("bottomDivider");
+				} else {
+					self._ui.header.classList.add("topDivider");
+					self._ui.footer.classList.add("bottomDivider");
+				}
+
+				eventUtils.trigger(content, "scrollupdate");
+				// }, false);
+			}
+
+			/**
+			 * Scroll event
+			 * @method _onScroll
+			 * @param {Element} content
+			 * @protected
+			 * @member ns.widget.core.Popup
+			 */
+			prototype._onScroll = function (content) {
```

Comment:
The same method above.

Reply:
This method is wrong. You should add listener in _bindEvents

Reply:
done.